src/Logging/Logger.php

```diff
@@ -0,0 +1,226 @@
+<?php
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Logging;
+
+use Google\Cloud\Logging\Connection\ConnectionInterface;
+
+/**
+ * A logger used to write entries to Google Stackdriver Logging.
+ */
+class Logger
+{
+    const EMERGENCY = 'EMERGENCY';
+    const ALERT = 'ALERT';
+    const CRITICAL = 'CRITICAL';
+    const ERROR = 'ERROR';
+    const WARNING = 'WARNING';
+    const NOTICE = 'NOTICE';
+    const INFO = 'INFO';
+    const DEBUG = 'DEBUG';
+
+    /**
+     * @var ConnectionInterface Represents a connection to Stackdriver Logging.
+     */
+    private $connection;
+
+    /**
+     * @var string The logger's formatted name to be used in API requests.
+     */
+    private $formattedName;
+
+    /**
+     * @var string The project ID associated with this logger.
+     */
+    private $projectId;
+
+    /**
+     * @param ConnectionInterface $connection Represents a connection to
+     *        Stackdriver Logging.
+     * @param string $name The name of the log to write entries to.
+     * @param string $projectId The project's ID.
+     */
+    public function __construct(ConnectionInterface $connection, $name, $projectId)
+    {
+        $this->connection = $connection;
+        $this->formattedName = "projects/$projectId/logs/$name";
+        $this->projectId = $projectId;
+    }
+
+    /**
+     * Deletes this log and all its log entries. The log will reappear if it
+     * receives new entries.
+     *
+     * Example:
+     * ```
+     * $logger->delete();
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/logging/docs/api/ref_v2beta1/rest/v2beta1/projects.logs/delete projects.logs delete API documentation.
+     * @codingStandardsIgnoreEnd
+     *
+     * @param array $options Configuration options.
+     */
+    public function delete(array $options = [])
+    {
+        $this->connection->deleteLog($options + ['logName' => $this->formattedName]);
+    }
+
+    /**
+     * Creates an entry which is to be written to the log.
```

Comment:
Could you add additional clarification here that you need to call `write()` to actually log anything?